src/filesystem/interface.js

```diff
@@ -1,30 +1,30 @@
-var { promisify } = require('es6-promisify');
+const { promisify } = require('es6-promisify');
 
-var Path = require('../path.js');
-var nop = require('../shared.js').nop;
+const Path = require('../path.js');
+const nop = require('../shared.js').nop;
 
-var Constants = require('../constants.js');
-var FILE_SYSTEM_NAME = Constants.FILE_SYSTEM_NAME;
-var FS_FORMAT = Constants.FS_FORMAT;
-var FS_READY = Constants.FS_READY;
-var FS_PENDING = Constants.FS_PENDING;
-var FS_ERROR = Constants.FS_ERROR;
-var FS_NODUPEIDCHECK = Constants.FS_NODUPEIDCHECK;
+const Constants = require('../constants.js');
```

Comment:
Let's use destructing assignment here, since we're pulling a bunch of stuff off `Constants`:

```js
const {
  FILE_SYSTEM_NAME,
  FS_FORMAT,
  ...
} = require('../constants.js');
```